content/en/docs/helm/helm_upgrade.md

```diff
@@ -39,6 +39,11 @@ set for a key called 'foo', the 'newbar' value would take precedence:
 helm upgrade [RELEASE] [CHART] [flags]
 ```
 
+To check the generated manifests of a release without upgrading the chart,
+use the '--dry-run' flag.
```

Comment:
use the '--dry-run' flag. --> the '--debug' and '--dry-run' flags can be combined.